imagersite/imager_images/tests.py

```diff
@@ -125,3 +128,125 @@ def test_user_can_see_its_photo_description(self):
         """Test the photo."""
```

Comment:
You forgot to change the name of the test. This is testing if an `Album` can see its photo description, not the `User`.